torch/_torch_docs.py

```diff
@@ -3286,6 +3286,41 @@ def merge_dicts(*dicts):
 
 """.format(**common_args))
 
+add_docstr(torch.igamma,
+           r"""
+igamma(input, other, *, out=None) -> Tensor
+
+Computes the regularized lower incomplete gamma function:
+
+.. math::
+    \text{out}_{i} = \frac{1}{\Gamma(\text{input}_i)} \int_0^{\text{other}_i} t^{\text{input}_i-1} e^{-t} dt
+
+where (1) :attr:`input` and :attr:`other` must be non-negative and (2)
+:attr:`input` and :attr:`other` must not be zero at the same index.
```

Comment:
The elements may not be at the same index due to broadcasting. I also think you can flip this sentence:

"when both :math:\`\\text{input}_i\` and :math:\`\\text{other}_i\` are weakly positive and at least one is strictly positive. If both are zero or either is negative then :math:\`\\text{out}_i=\\text{nan}`.

Reply:
I'm not sure how to express the index to represent the broadcasting. I looked at `torch.hypot` documentation, it also puts the same indices for both operands. I think users who know broadcast should understand what it really means.

Reply:
I agree that _i is clear because it suggests the correspondence. I meant to say that I think saying the elements have the same index, like the current expression does, is misleading. I think you're right that most readers familiar with PyTorch would know what the current documentation means, but we should strive to be as clear as possible. A lot of our code and documentation is copy-pasted, after all, so imperfections have a habit of multiplying.